### PR TITLE
Include Keybase in Finder favorites

### PR DESCRIPTION
**Pull request by gabriel**
We can't reliably create a file list (Finder) favorite directly for /keybase, since it is a remote volume mount, and there is some funkiness there (like the URL property not resolving, or the display name being ignored). If we create a symlink though, all these problems are avoided. So we'll create a symlink to /keybase and add this as the file list favorite item.

Also I'm changing project file to use Automatic Provisioning. We build and sign using a script, and we verify the signing identity in the install scripts, so this change doesn't affect our builds and makes it easier to test directly from Xcode.

### CODE REVIEW

File: go/launchd/launchd.go [hunk 1]
```diff
@@ -168,6 +168,10 @@ func (s Service) Restart(wait time.Duration) error {
 
 // WaitForStatus waits for service status to be available
 func (s Service) WaitForStatus(wait time.Duration, delay time.Duration) (*ServiceStatus, error) {
+	if wait <= 0 {
```
Comment (gabriel) [inline go/launchd/launchd.go]: Fixing bug where nil status was being returned if wait = 0.
This wasn't causing any issues though, I just noticed it cause things to be reported wrong in the logs abount the install status for services.

**Comment by gabriel** [comment]
@chromakode can review?
cc @keybase/react-hackers

**Comment by gabriel** [comment]
bump

File: osx/KBKit/KBKit/Component/KBMountDir.m [hunk 2]
```diff
@@ -94,6 +100,55 @@ - (void)uninstall:(KBCompletion)completion {
...
+  NSString *symPath = [config appPath:@"Keybase" options:0];
+  if (![[NSFileManager defaultManager] fileExistsAtPath:symPath]) {
+    if ([[NSFileManager defaultManager] createSymbolicLinkAtPath:symPath withDestinationPath:config.mountDir error:error]) {
+      return NO;
+    }
+  }
+
+  NSURL *URL = [NSURL fileURLWithPath:symPath];
+  NSString *name = [config appName];
+  //DDLogDebug(@"File list favorite items: %@", [KBSharedFileList debugItemsForType:kLSSharedFileListFavoriteItems]);
```
Comment (chrisnojima) [inline osx/KBKit/KBKit/Component/KBMountDir.m]: 🔥

**Pull request merged**